shared/js/background/chrome-events.es6.js

```diff
@@ -716,9 +723,10 @@ if (chrome.webRequest.OnBeforeSendHeadersOptions.EXTRA_HEADERS) {
 chrome.webRequest.onBeforeSendHeaders.addListener(
     request => {
         const GPCHeader = GPC.getHeader()
+        const GPCEnabled = tdsStorage.config.features?.gpc.state === 'enabled'
```

Comment:
We are doing this check quite a lot (`tdsStorage.config.features?.gpc.state`) - can we move it to a helper function e.g.`isFeatureEnabled('name') => bool` ?